mobify/platform-scaffold
# [IM to SDK] top level results

This PR removes the home and custom branches from the scaffold Integration manager. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-21
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/693

## Changes
- Removed top level results from the Integration Manager
- Updated paths for IM top level results to point to the SDK versions

## How to test-drive this PR
- checkout the im-top-level-results branch for the SDK
- cd into the SDK folder
- run `npm link`
- run `npm run dev:build`
- cd back to the scaffold
- run `npm link progressive-web-sdk`
- run `npm run dev`
- preview the merlins site
- the home page should load
- open the navigation
- it should be populated with data
